source/slang/lookup.cpp

```diff
@@ -297,6 +297,23 @@ void DoLocalLookupImpl(
                 session,
                 name, extDeclRef, request, result, inBreadcrumbs);
         }
+
+    }
+    // for interface decls, also lookup in the base interfaces
```

Comment:
Why only for interfaces?

Reply:
Because we are supporting implementation inheritance yet, and I don't want to deal with potential problems when a lookup on a concrete type returns a decl defined in the base interface type...